sdk/storage/azure-storage-blob/azure/storage/blob/_shared/base_client_async.py

```diff
@@ -11,10 +11,7 @@
 import logging
 
 from azure.core.pipeline import AsyncPipeline
-try:
```

Comment:
same as above file.

Reply:
This change was already made in files and queues.